### PR TITLE
Use `@netlify/eslint-config-node`

### PR DESCRIPTION
**Pull request by ehmicky**
This adds `@netlify/eslint-config-node`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,34 +73,29 @@
...
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.4",
     "execa": "^3.4.0",
     "get-stream": "^5.1.0",
     "husky": "^4.2.5",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.0.0",
-    "path-exists": "^4.0.0",
-    "prettier": "^1.19.1",
```
Comment (ehmicky) [inline package.json:92]: `common-path-prefix` and `path-exists` are already production dependencies.

File: package.json [hunk 2]
```diff
@@ -65,34 +73,29 @@
     "yargs": "^15.4.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.5",
-    "@babel/plugin-transform-runtime": "^7.10.5",
-    "@babel/preset-env": "^7.10.4",
-    "@babel/runtime": "^7.10.5",
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
```
Comment (erezrokah) [inline package.json:77]: I think we can remove `@commitlint/cli` and `@commitlint/config-conventional` too.

**Review by erezrokah — APPROVED** [review]
🚀

File: package.json (hunk 2 above)
Comment (ehmicky) [reply package.json:77]: Done :+1:

**Pull request merged**